### PR TITLE
vendor: update test-infra dependency

### PR DESCRIPTION
**Pull request by AlexNPavel**
Update test-infra dependency (and fix `update-vendor` make target).

/cc @stevekuznetsov

### CODE REVIEW

**Comment by GeoBK** [comment]
lgtm

File: pkg/backporter/backporter_test.go [hunk 1]
```diff
@@ -138,6 +138,7 @@ func TestGetClonesHandler(t *testing.T) {
 			data: ClonesTemplateData{
 				clone,
 				[]*bugzilla.Bug{
+					clone,
```
Comment (GeoBK) [inline pkg/backporter/backporter_test.go:141]: clone will have to be after toBeCloned - as the order matters now

**Comment by petr-muller** [comment]
/hold

Apparently a genuine failure?

**Comment by AlexNPavel** [comment]
> /hold
> 
> Apparently a genuine failure?

Yep, there's a PR in test-infra that will fix this. Will update this PR when that merges.

**Pull request merged**